Sources/Twift+Tweets.swift

```diff
@@ -262,6 +262,13 @@ public struct DeleteResponse: Codable {
   public let deleted: Bool
 }
 
+/// A response object pertaining to list created
```

Comment:
This type definition should probably live in `Twift+Lists.swift` instead of `Twift+Tweets.swift`

Reply:
Oh wow, did not even pay attention to the file I was in. Just saw deletedResponse and thought they would fit nicely together 😂